video_store.py

```diff
@@ -0,0 +1,146 @@
+import requests
+from datetime import datetime
+
+class VideoStore:
+    def __init__(self, url="https://retro-video-store-api.herokuapp.com", selected_video=None, selected_customer=None):
+        self.url = url
+        self.selected_video = selected_video
+        self.selected_customer = selected_customer
+        
+
+##### VIDEO #####
+    def create_video(self, title="Default Title", release_date="Default Date", total_inventory=0):
+        query_params = {
+            "title": title,
+            "release_date": release_date,
+            "total_inventory": total_inventory
+        }
+        response = requests.post(self.url+"/videos",json=query_params)
+        return response.json()
+
+    def update_video(self, title=None, release_date="Default Date", total_inventory=0):
+        if not title:
+            title = self.selected_video["title"]
+        if not release_date:
+            release_date = self.selected_video["release_date"]
+        if not total_inventory:
+            total_inventory = self.selected_video["total_inventory"]
+
+        query_params = {
+        "title": title,
+        "release_date": release_date,
+        "total_inventory": total_inventory}
+
+        response = requests.put(self.url+f"/videos/{self.selected_video['id']}", 
+                                json=query_params)
+        print("response:", response)
+        self.selected_video = response.json()["video"]
+        return response.json()    
+
+    def delete_video(self):
```

Comment:
This function relies on the menu to ensure that a video has been selected.  When writing a class like this I recommend adding error checking to verify that selected_video is not None and has a field 'id' accessing 'id', or wrapping this section in a try/except block to catch a potential error.